kivy/config.py

```diff
@@ -932,6 +938,7 @@ def name(self, value):
 
         elif version == 25:
             Config.setdefault('graphics', 'always_on_top', '0')
+            Config.setdefault("graphics", "show_taskbar_icon", "1")
```

Comment:
Better to require another config migration, even if we bumped to `version = 26` just yesterday.

Reply:
Ok, adressed!